geo/src/utils.rs

```diff
@@ -93,9 +93,6 @@ pub fn partial_min<T: PartialOrd>(a: T, b: T) -> T {
     }
 }
 
-// Moved to their own module, but we re-export to avoid breaking the API.
-pub use crate::coordinate_position::{coord_pos_relative_to_ring, CoordPos};
```

Comment:
The `util` module is not publicly accessible, so there's no need to worry about breaking an API's **within** it.